rokwire.yaml

```diff
@@ -333,6 +335,108 @@ paths:
                   $ref: '#/components/schemas/Notification'
         401:
           description: Unauthorized
+  /events:
+    get:
+      tags:
+      - Events
+      summary: searches events
+      description: By passing in the appropriate options, you can search for events matching your criteria in the system
+      operationId: searchEvents
+      parameters:
+      - name: searchString
+        in: query
+        description: pass an optional search string for looking up events
+        required: true
+        style: form
+        explode: true
+        schema:
+          type: string
+      - name: skip
+        in: query
+        description: number of records to skip for pagination
+        required: false
+        style: form
+        explode: true
+        schema:
+          minimum: 0
+          type: integer
+          format: int32
+      - name: limit
+        in: query
+        description: maximum number of records to return
+        required: false
+        style: form
+        explode: true
+        schema:
+          maximum: 50
+          minimum: 0
+          type: integer
+          format: int32
+      responses:
+        200:
+          description: search results matching criteria
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Events'
+
+        401:
+          description: Unauthorized
+        404:
+          description: Profile not found
+        500:
+          description: Internal error
+    post:
+      tags:
+      - Events
+      summary: create an event
+      description: create an event record into the system
+      operationId: createEvent
+      requestBody:
+        description: Created event object
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Events'
+        required: true
+      responses:
+        201:
+          description: Events created
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Events'
+        400:
+          description: Bad request
+        401:
+          description: Unauthorized
+        500:
+          description: Internal error
+  /events/{eventId}:
+    delete:
```

Comment:
I am not sure what exactly the requirement for events is. But is mobile app user allowed to create/delete an event? I

Reply:
The mobile app will not, but there will be an independent client program (web-based or CLI based), which will help us manage events (populating, updating, deleting, etc.)